monai/networks/blocks/patchembedding.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import math
+from typing import Tuple, Union
+
+import torch
+import torch.nn as nn
+
+from monai.utils import optional_import
+
+einops, has_einops = optional_import("einops")
+
+
+class PatchEmbeddingBlock(nn.Module):
```

Comment:
could we add 2d support as well?

Reply:
Sure, how about adding this in a separate feature request  ?